src/sql-parser/src/ast/mod.rs

```diff
@@ -694,6 +698,17 @@ impl fmt::Display for Connector {
                 }
                 Ok(())
             }
+            Connector::AvroObject { path, with_options } => {
+                write!(
+                    f,
+                    "AVRO OBJECT '{}'",
```

Comment:
The specification name for these files is `Object Container Files` ([spec](https://avro.apache.org/docs/1.8.1/spec.html#Object+Container+Files)), so I think this should either be `OCF` or `OBJECT CONTAINER FILE`. Both of which I am sorry for suggesting :-/

Reply:
I would really rather not require five keywords in a row (`FROM AVRO OBJECT CONTAINER FILE`) so let's go with the abbreviated `OCF`)